<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Generatore cartellino identificativo AEE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>701</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cartellino</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>440</width>
        <height>400</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Input</string>
      </property>
      <widget class="QLineEdit" name="revInput">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Product number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pnInput">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>190</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>80</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Revisione</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>110</width>
         <height>27</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="qtyInput">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>90</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Destinazione</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>70</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Quantità</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Data</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>326</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Operatore</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>212</y>
         <width>130</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Riferimento ordine</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="refInput">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>232</y>
         <width>380</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="destInput">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>380</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="operatorInput">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>350</y>
         <width>380</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>231</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>logo</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>250</y>
        <width>241</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Oputput</string>
      </property>
      <widget class="QPushButton" name="resetBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="genBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Genera</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Shipping list</string>
     </attribute>
     <widget class="QTableWidget" name="shippingTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>541</width>
        <height>341</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Codice</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantità</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N° buste</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="clearTableBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancella tabella</string>
      </property>
     </widget>
     <widget class="QPushButton" name="undoBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Annulla inserimento</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>360</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Esporta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="listInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>540</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pnInput</tabstop>
  <tabstop>revInput</tabstop>
  <tabstop>qtyInput</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>refInput</tabstop>
  <tabstop>destInput</tabstop>
  <tabstop>operatorInput</tabstop>
  <tabstop>resetBtn</tabstop>
  <tabstop>genBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
